lib/services/beanstalk/deployable-artifact.js

```diff
@@ -101,10 +161,10 @@ function prepareAndUploadWar(pathToArtifact, ebextensionsToInject) {
     return Promise.reject(new Error("Not Implemented"));
 }
 
-function prepareAndUploadJar(ownServiceContext, pathToArtifact, ebextensionsToInject) {
+function prepareAndUploadJar(accountConfig, ownServiceContext, pathToArtifact, ebextensionsToInject) {
```

Comment:
Rather than passing accountConfig around, you can just require it at the top of your file like this:

```
const accountConfig = require('../../common/account-config')().getAccountConfig();
```

Then you can use it in any function in the file that needs it.